src/utils/fetchclient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing the `eslint-disable` line if possible and fix any potential TypeScript issues instead of disabling the rule 🧐